09_timer/timer.rb

```diff
@@ -2,15 +2,22 @@
 
 class Timer
   attr_accessor :seconds
-  def initialize(seconds = 0)
-    @seconds = seconds
+
+  def initialize
+    @seconds = 0
   end
 
   def time_string
-  result = []
-  hour 
-  minutes
-  result.join
+    seconds = @seconds % 60
+    minutes = (@seconds / 60) % 60
+    hours = @seconds / (60**2)
```

Comment:
60 and 2 - magic numbers, spaces around operator
also do not forget about `attr_accessor` that is called at the beginning of a class => avoid using instance (@) variables directly.